src/Player.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Player.css";
+import Avatar from "@material-ui/core/Avatar";
+
+function Player({name, image, position, FFP}) {
+    
+ return (
+
+    <div className="player">
+        <div className="playerHeader">
+        <Avatar 
+        className="playerAvatar"
+        alt={"name"}
```

Comment:
We should remove the quotes here. Currently it's just setting the alt tag to the string 'name'

Reply:
Resolved